pkg/maps/ipcache/ipcache.go

```diff
@@ -124,14 +123,13 @@ func NewKey(ip net.IP, mask net.IPMask, clusterID uint8) Key {
 type RemoteEndpointInfo struct {
 	SecurityIdentity uint32     `align:"sec_identity"`
 	TunnelEndpoint   types.IPv4 `align:"tunnel_endpoint"`
-	NodeID           uint16     `align:"node_id"`
```

Comment:
Do we need to patch the whole here with `Pad uint16` then when upgrade/downgrade we don't have to adjust the maps formats? Seems this just pushes the pad to the end.